### PR TITLE
cotw and notable changes for TWIR 127

### PR DESCRIPTION
**Pull request by llogiq**
The same procedure as every week. :smile:

### CODE REVIEW

File: drafts/2016-04-18-this-week-in-rust.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This week's edition was edited by: [Vikrant](https://github.com/nasa42) and [llo
 
 # Crate of the Week
 
-This week's Crate of the Week is [gcc](https://crates.io/crates/gcc), a crate to easily use the local C compiler, which makes FFI with a build script a breeze. Thanks to [Ulrik Sverdrup](https://users.rust-lang.org/users/bluss) for the suggestion!
+This week's Crate of the Week is [owning_ref](https://crates.io/crates/owning_ref), which contains a reference type that can carry it's owner with it. Thanks to [Diwic](https://users.rust-lang.org/users/diwic) for the suggestion!
```
Comment (durka) [inline drafts/2016-04-18-this-week-in-rust.md]: its

Comment (llogiq) [inline drafts/2016-04-18-this-week-in-rust.md]: Thanks.

**Comment by nasa42** [comment]
Thanks!

**Comment by llogiq** [comment]
You're welcome. Also welcome back :smile:.

**Pull request merged**